docs/rfcs/1013-parsec-account.md

```diff
@@ -505,12 +498,13 @@ Anonymous account API:
 ```
 
 > [!NOTE]
-> Since the request is authenticated with the token field it contains, it could
+> Since the request is authenticated with the validation code field it contains, it could
```

Comment:
```suggestion
> Since the request is authenticated with the validation code and the email field it contains, it could
```

I'm not sure this part is still relevant though